LiteDB/Client/Storage/LiteFileStream.cs

```diff
@@ -23,6 +24,7 @@ public partial class LiteFileStream<TFileId> : Stream
         private byte[] _currentChunkData = null;
         private int _positionInChunk = 0;
         private MemoryStream _buffer;
+        private Dictionary<int, long> _chunkLengths = new Dictionary<int, long>();
```

Comment:
Not trying to force you change this, but are you sure sure separating fields from their main use is a good idea? I mean if it would be used by many different partials, this would imo the best place but you are only using it in the ``.Read`` partial. 

What do you think? Does that make sense?